src/test/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/integration/DeleteVisitByReferenceTest.kt

```diff
@@ -48,7 +48,7 @@ class DeleteVisitByReferenceTest : IntegrationTestBase() {
       .save()
 
     visitContactCreator(visit = visitCC, name = "Jane Doe", phone = "01234 098765")
-    visitVisitorCreator(visit = visitCC, nomisPersonId = 123L)
+    visitVisitorCreator(visit = visitCC, nomisPersonId = 123L, visitContact = true)
```

Comment:
Are there any tests needed where `visitContact` is `false`?

Reply:
Defiantly needed, we needed to check the values on assert also for update and create... good spot